commands/livecmd.go

```diff
@@ -64,7 +64,7 @@ func GetLiveCommand(name string, f util.Factory) *cobra.Command {
 	applyCmd.Long = livedocs.ApplyShort + "\n" + livedocs.ApplyLong
 	applyCmd.Example = livedocs.ApplyExamples
 
-	previewCmd := preview.GetPreviewRunner(f, ioStreams).Command
+	previewCmd := preview.PreviewCommand(f, ioStreams)
```

Comment:
Let's keep the `Runner`. We will need it to inject the `ResourceGroup` versions of structures with the `ResourceGroupProvider`.

Reply:
But we can just update that when we have the provider for ResourceGroup ready. This already is using the default provider.